src/core/hw/gfxip/gfx9/gfx9CmdUtil.cpp

```diff
@@ -3955,12 +3964,7 @@ size_t CmdUtil::BuildWaitRegMem64(
     pPacket->ordinal8.mask_hi                 = HighPart(mask);
     pPacket->ordinal9.u32All                  = 0;
     pPacket->ordinal9.bitfields.poll_interval = Pal::Device::PollInterval;
-
-    if (Pal::Device::EngineSupportsGraphics(engineType))
-    {
-        pPacket->ordinal2.bitfields.engine_sel = static_cast<ME_WAIT_REG_MEM64_engine_sel_enum>(engine);
-    }
-    else
+    if (!Pal::Device::EngineSupportsGraphics(engineType))
```

Comment:
```suggestion
    if (Pal::Device::EngineSupportsGraphics(engineType) == false)
```

Reply:
Done

Reply:
It seems github doesn't like manually doing the suggested change. I made the change you suggested into my branch and force pushed. So it is ready for review again.